variants/deploy_with_ackama_ec2_capistrano/template.rb

```diff
@@ -0,0 +1,228 @@
+puts "=" * 80
+puts "Starting variant: deploy_with_ackama_ec2_capistrano"
+puts "=" * 80
+
+copy_file "variants/deploy_with_ackama_ec2_capistrano/aws_ec2_environment.yml", "config/aws_ec2_environment.yml"
+
+append_to_file("Gemfile") do
+  <<~'EO_RUBY'
+
+    # Deployment
+    #
+    # We tell bundler not to require these gems by default because the code that
+    # requires them will explicitly require them.
+    #
+    gem "capistrano", require: false
+    gem "capistrano-bundler", require: false
+    gem "capistrano-rails", require: false
+    gem "capistrano-rbenv", require: false
+    gem "capistrano-rake", require: false
+    gem "ed25519", require: false
+    gem "bcrypt_pbkdf", require: false
+    gem 'aws_ec2_environment', git: 'https://github.com/ackama/aws_ec2_environment.git'
```

Comment:
You can use `github` here:

```suggestion
    gem "aws_ec2_environment", github: "ackama/aws_ec2_environment"
```

This has the slight (dis)advantage of cloning using `https` rather than `ssh` which technically is less optimal but if you're using `ssh` then you need to authenticate even if the repo is public unless you're in the organisation; for local dev this shouldn't be an issue, but comes up in CI (I hit this in a certain pastry themed project that we're migrating to AWS, that lives in its own org owned by the client).

(I am hoping to publish the gem soon)